### DISCUSSION TITLE
Shipping Confirmation: Update shipping and tracking information for the orders

### DISCUSSION THREAD

**Discussion by hardikradixweb** [amazon-php/sp-api-sdk#233] [Q&A]
Hello,

I have successfully implemented the orders pull system where I can pull orders and order items, but now I have to update shipping information to those orders I have tried to understand the documentation but there are many shipping APIs that amazon provides, so do anyone have any idea about how I can update shipping information (USPS, FedEx, UPS) to the amazon orders? and by which API i can achieve this?

Thanks in the Advance!!

**Comment by johncarlson21** [comment]
I am also at this point and I feel that docs are terrible for Amazon. :|
I need to figure out how to properly create the "package_details"
would be nice if this library had examples of how to build this section as it looks like you need to make multiple calls to other models to make this work correctly. ( package_details, order_items ) etc.
